policyengine_us/variables/household/demographic/person/hi_child_support_received.py

```diff
@@ -0,0 +1,8 @@
+from policyengine_us.model_api import *
+
+
+class hi_child_support_received(Variable):
+    value_type = float
+    entity = Person
+    label = "Child receiving support"
+    definition_period = YEAR
```

Comment:
```suggestion
    definition_period = YEAR
    defined_for = state_code.HI
```

Reply:
# Obtain the amount of all support
        child_support = person("child_support_received", period)
        # Test if the child receive more than half support from public agency(tanf or social_security_survivor_benefits)
        government_payments = person("social_security_survivors", period) + person.spm_unit("tanf", period)
        public_support_over_half = (
            government_payments / child_support
        ) > p.minor_child.support_proportion_threshold